.buildkite/go/build.sh

```diff
@@ -29,6 +29,10 @@ pushd go
   make all GO_BUILD_E2E_COVERAGE=1
 popd
 
-pushd tests/upgrade
-  make all
+pushd tests/upgrade/pre
+  OASIS_GO=go1.19.10 make all
 popd
+
+pushd tests/upgrade/post
+  OASIS_GO=go1.20.2 make all
```

Comment:
Will this change to 1.21 with this bump or what is the plan?

Reply:
I will bump to 1.21 once this PR is merged to master.